.github/workflows/text-linter.yml

```diff
@@ -1,19 +0,0 @@
-name: text-linter
-on:
-  push:
-  pull_request:
-    branches:
-      - master
-    types: [opened, synchronize, reopened, ready_for_review]
-
-jobs:
-  lint:
-    runs-on: ubuntu-latest
-    if: github.event.pull_request.draft == false
-    steps:
-      - uses: actions/checkout@v3
-      - uses: DavidAnson/markdownlint-cli2-action@v9
-        with:
-          globs: |
-            *.md
-            !**/*.md
```

Comment:
Why is the action deleted?

Reply:
Strange, it is showing as a rename now but however I was viewing the PR before I did not see that

Reply:
i renamed it from `text-linter` to `markdown-linter` to be more descriptive.